hypertrace-alert-engine/src/integrationTest/resources/notification-channels-2.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "channelName": "channel-1",
+    "channelId": "channel-id-1",
```

Comment:
Shouldn't channel id be 2 ? (also in dynamic alert rule)

Reply:
Yes, both files are having the same channel id. So, we can remove the one channel file, and keep only one `notification-channels.json`
Or, We can change `"channelId": "channel-id-1",` in one of the rule files.

Reply:
doesn't matter